### PR TITLE
Create aml-kyc-requirements.md

### PR DESCRIPTION
**Pull request by einotna**

### CODE REVIEW

File: about/aml-kyc-requirements.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+layout: page
+title: AML/KYC Requirements
+excerpt: "What the Foundation is required to collect for compliance"
+---
+
+Last updated: December 2, 2020.
+
+
+_Compliance_:  **To execute a transfer of funds, the Zcash Foundation is legally required to obtain the following information from you: [Privacy guarantee]**
```
Comment (mpguerra) [inline about/aml-kyc-requirements.md:10]: Seems like there is a missing link to the Privacy guarantee here

File: about/aml-kyc-requirements.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+_Disbursement_: Your funds will be disbursed in a shielded Zcash cryptocurrency (ZEC), then it will be via an on-blockchain fund transfer transaction. The Foundation will use this third-party service and market for converting ZEC to other currencies are listed here based on the price of the agreed-upon date close of day at : [https://messari.io/asset/zcash](https://messari.io/asset/zcash). For all grants, the agreed-upon date will be the date that the grant was approved, as noted in the grant platform. Note that the Zcash Foundation understands the regulatory and compliance risks associated with transacting in cryptocurrencies. and 
+
```
Comment (mpguerra) [inline about/aml-kyc-requirements.md:17]: hanging "and" after the last sentence. Maybe incomplete?

**Review by mpguerra — COMMENTED** [review]
I think the last paragraph needs reviewing, I'm finding it hard to parse.

File: about/aml-kyc-requirements.md (hunk 1 above)
Comment (einotna) [reply about/aml-kyc-requirements.md:10]: it's some legalese punctuation, it's not supposed to be a link

File: about/aml-kyc-requirements.md (hunk 2 above)
Comment (einotna) [reply about/aml-kyc-requirements.md:17]: good catch. removed.

**Review by mpguerra — APPROVED** [review]
